### PR TITLE
Allow more aspects of the base template to be overridden

### PR DESCRIPTION
**Pull request by dave-irvine**

### CODE REVIEW

File: views/pulsar/layouts/base.html.twig [hunk 1]
```diff
@@ -63,45 +63,48 @@
...
+                {% block search %}
                 <input type="search" name="q" id="case-search" value="" placeholder="Search" class="main-search__field" autocomplete="off">
                 <button class="main-search__submit" type="submit" name="case-search"><i class="icon-search"></i></button>
+                {% endblock %}
             </form>
 
             <div class="content-main">
                 <div class="tabs__content">
                     <nav class="breadcrumb">
+                    {% block breadcrumb %}
```
Comment (Stanton) [inline views/pulsar/layouts/base.html.twig]: Will improve this shortly with #203

File: views/pulsar/layouts/base.html.twig [hunk 2]
```diff
@@ -63,45 +63,48 @@
...
+                {% endblock %}
                 </div>
 
-                {% include '@pulsar/lexicon/includes/notifications.html.twig' %}
+                {% block notifications %}{% endblock %}
 
             </div>
 
             <form action="/search" class="main-search" data-ui="search-box">
+                {% block search %}
```
Comment (Stanton) [inline views/pulsar/layouts/base.html.twig]: Is the `/search` form action always appropriate, should we include `<form>` within the block too?

Comment (dave-irvine) [inline views/pulsar/layouts/base.html.twig]: It becomes tricky for consistency :| All the other blocks sit inside the wrapper element, but in this case you're right, it needs to be outside. Unless we add a `div` wrapper that we can put the block inside?

Comment (Stanton) [inline views/pulsar/layouts/base.html.twig]: It's not a structural element so I'd say putting the entire `<form>` within the block should be ok in this case

**Pull request merged**